libs/mock-block-dock/src/version.ts

```diff
@@ -1 +1 @@
-export const MOCK_BLOCK_DOCK_VERSION = "0.0.38";
+export const MOCK_BLOCK_DOCK_VERSION = "0.0.39";
```

Comment:
This change happened whilst running `yarn install`, is that expected? cc @CiaranMn @kachkaev

Reply:
Yes – you can commit it.

It's a `codegen` action we need to sort out at some point.